source/install/cloud-changelog.md

```diff
@@ -4,12 +4,39 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-05-11](#release-2022-05-11)
 - [Release 2022-04-28](#release-2022-04-28)
 - [Release 2022-04-13](#release-2022-04-13)
 - [Release 2022-03-30](#release-2022-03-30)
 - [Release 2022-03-16](#release-2022-03-16)
 - [Release 2022-03-08](#release-2022-03-08)
-- [Release 2022-03-02](#release-2022-03-02)
+
+## Release 2022-05-11
+
+### Improvements
+
+#### User Interface (UI)
+ - For toggling the channel information in the right-hand pane, a shortcut CTRL/CMD+ALT+I was added.
+ - Added an "Unread Channels" section to the channel switcher and included "Threads" in the results.
+
+#### Performance
```

Comment:
I don't think we need to include server performance related stuff in the cloud changelog, since we manage the servers users or admins don't feel it